hack/testing/check-EFK-running.sh

```diff
@@ -18,6 +18,58 @@ function waitFor() {
   return 1
 }
 
+function checkESStarted() {
+
+  local pod=$1
+  local cluster_service
+
+  if ! cluster_service=$(waitForValue "oc logs $pod | grep '\[cluster\.service[[:space:]]*\]'"); then
```

Comment:
This is causing the test failures::

    ./check-EFK-running.sh: line 26: waitForValue: command not found

Where is `waitForValue()` defined?